editor/plugins/game_view_plugin.cpp

```diff
@@ -894,12 +918,16 @@ GameView::GameView(Ref<GameViewDebugger> p_debugger, WindowWrapper *p_wrapper) {
 	suspend_button->set_theme_type_variation(SceneStringName(FlatButton));
 	suspend_button->connect(SceneStringName(toggled), callable_mp(this, &GameView::_suspend_button_toggled));
 	suspend_button->set_tooltip_text(TTR("Suspend"));
```

Comment:
Looks like tooltip text needs to be removed, because it duplicates the shortcut.
![image](https://github.com/user-attachments/assets/ccf68be6-257b-4630-bf19-147b8a62858b)